src/main/java/com/microsoft/sqlserver/jdbc/SQLServerClob.java

```diff
@@ -91,7 +88,9 @@ final JDBCType getJdbcType() {
     private ArrayList<Closeable> activeStreams = new ArrayList<>(1);
 
     transient SQLServerConnection con;
-    private static Logger logger;
+    
+    private final Logger logger = Logger.getLogger(getClass().getName());
```

Comment:
How about this comment that got deleted?
`// Loggers should be class static to avoid lock contention with multiple threads`

Reply:
Well - its because this logger object won't be static in this case.